.github/workflows/python-sdk-publish.yaml

```diff
@@ -10,6 +10,9 @@ jobs:
     name: Publish new version
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
```

Comment:
Can you propose fixes not related to this PR with other PR?

Also I believe it's need for SDK publication, which needs to be published, so we have status clear on this PR for merging

Reply:
I've created https://github.com/serverless/console/pull/466 to isolate those in a separate PR. I'll update this one to clean them and will let you know.